package.json

```diff
@@ -60,7 +60,8 @@
   "lint-staged": {
     "actions/**/*.+(js|json)": [
       "npm run update-actions-build"
-    ]
+    ],
+    "*.+(js/json) ./spec/**/*.+(js/json) ./lib/*.js ./fixtures/*.js ./actions_build/*.js actions/**/*.+(js|json) ": "npm run lint"
```

Comment:
We do not need to lint `actions_build/` since that folder is auto-generated.

Reply:
removed